CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Removed timestamp from AutoMod messages. (#3503)
```

Comment:
```suggestion
```
Old and Duplicate changelog.

Reply:
Yeah, i don't know how I can remove these from the branch.

Reply:
> Yeah, i don't know how I can remove these from the branch.

By deleting them in the file and pushing those changes??

Reply:
> Yeah, i don't know how I can remove these from the branch.

You click the button on felanbird's comment to merge the change he suggested.